src/lava/lib/optimization/apps/scheduler/utils/util_funcs.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+
+def is_visible(satellite_y, request_y, view_height):
```

Comment:
I think this same functionality has been implemented in the Satellite Scheduling Problem, isn't it?
Same for is_same_satellite. Maybe best to check with redundancy there.

Reply:
Yup. This entire file should have been removed.